pkg/pgmodel/querier/query_builder.go

```diff
@@ -376,16 +410,18 @@ func buildTimeSeries(rows []timescaleRow, lr lreader.LabelsReader) ([]*prompb.Ti
 			promLabels = append(promLabels, prompb.Label{Name: label.Name, Value: label.Value})
 
 		}
-
-		if schemaLabelName, schemaLabelValue := getSchemaLabel(row.schema); schemaLabelName != "" {
-			// If its a custom schema, we need to update the metric name as well.
+		if row.metric != "" {
```

Comment:
if row.metric is only for when metrics need to be overriden maybe change name to row.metricOverride?